mlir/lib/Dialect/MIOpenOps/LowerMIOpenOps.cpp

```diff
@@ -589,19 +589,20 @@ struct Conv2DOpRewritePattern : public OpRewritePattern<miopen::Conv2DOp> {
     auto rightPadW = wiPadded > (leftPadW + wi) ? wiPadded - (leftPadW + wi) : 0;
  
     // Set attributes for gridwise_gemm op.
-    llvm::SmallVector<NamedAttribute, 8> gridwiseGemmAttrs {
-      b.getNamedAttr("filter_layout", filterLayoutAttr),
-      b.getNamedAttr("filter_dimension", b.getI64ArrayAttr(filterShape)),
-      b.getNamedAttr("input_layout", inputLayoutAttr),
-      b.getNamedAttr("input_dimension", b.getI64ArrayAttr(inputShape)),
-      b.getNamedAttr("output_layout", outputLayoutAttr),
-      b.getNamedAttr("output_dimension", b.getI64ArrayAttr(outputShape)),
-      b.getNamedAttr("dilations",  dilationsAttr),
-      b.getNamedAttr("strides", stridesAttr),
-      b.getNamedAttr("padding", b.getArrayAttr({
-        paddingAttr,
-        b.getI32ArrayAttr({rightPadH, rightPadW})
-      })),
+    llvm::SmallVector<NamedAttribute, 8> gridwiseGemmAttrs{
+        b.getNamedAttr("kernel_algorithm", b.getStringAttr("v4r4")),
```

Comment:
this should be submitted in a later PR where lowering logic is amended. it has nothing to do with is particular PR which introduces miopen.conv2d_bwd_data op.

Reply:
Will remove